modules/installation-initializing.adoc

```diff
@@ -332,6 +332,12 @@ ifdef::vsphere,vmc[]
 +
 The installation program connects to your vCenter instance.
 ... Select the data center in your vCenter instance to connect to.
++
+[NOTE]
+====
+After you create the installation configuration file, you can modify the file to create an environment where you can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. For more information about creating this environment, see the section named _VMware vSphere region and zone enablement_.
```

Comment:
The ISG specifies keeping sentences to 32 words or fewer. Please break this up into smaller sentences.